filament/src/MaterialParser.h

```diff
@@ -187,6 +191,7 @@ class MaterialParser {
     filaflat::ChunkContainer& getChunkContainer() noexcept;
     filaflat::ChunkContainer const& getChunkContainer() const noexcept;
     MaterialParserDetails mImpl;
+    mutable std::atomic<std::optional<uint32_t>> mCrc32;
```

Comment:
I don't like this because this doesn't gurantee that mCrc32 won't use a lock. it may or may not.
if you need an atomic, just store the crc and use 0 to mean it's not there.